scripts/publish.sh

```diff
@@ -724,8 +724,10 @@ if [ "${1}" = "cicd-verify" ]; then
 			verify_ecr ${region} ${classic_regions_account_id} true
 		done
 	elif [ "${2}" = "stable" ]; then
-		verify_dockerhub stable
-		verify_public_ecr stable
+		if [ "${3}" = "us-west-2" ]; then
+			verify_dockerhub stable
+			verify_public_ecr stable
+		fi
```

Comment:
I see the same two functions being called with the same arguments- how does this change anything?

Reply:
several days ago, we has this commit: https://github.com/aws/aws-for-fluent-bit/commit/5eb9a3fb37ca6bc550b76587295fefa6f0397b29. It adds dockerhub login in `verify_dockerhub()`. In this case, we can only call `verify_dockerhub()` in our main account. In our other accounts like BAH, it cannot log into our dockerhub as it doesn't have the creds. Therefore, we need to add a check here that we only call it when we are in `us-west-2`

Reply:
Oh I see. You are only doing these steps in us-west-2 because that the only stage where we verify public ECR/Dockerhub?

Reply:
So basically, previously we had a bug because we were verifying docker hub in all regions, even though we only need to do it once??